CHANGELOG.md

```diff
@@ -16,7 +16,7 @@
 - Do not drop sessions due to an invalid age constraint set to `0`. ([#855](https://github.com/getsentry/relay/pull/855))
 - Do not emit outcomes after forwarding envelopes to the upstream, even if that envelope is rate limited, rejected, or dropped. Since the upstream logs an outcome, it would be a duplicate. ([#857](https://github.com/getsentry/relay/pull/857))
 - Fix status code for security report. ([#864](https://github.com/getsentry/relay/pull/864))
-- Add missing fields for ExpectCT security reports. ([#865](https://github.com/getsentry/relay/pull/865))
+- Support missing fields and directives in CSP reports. ([#865](https://github.com/getsentry/relay/pull/865), [#876](https://github.com/getsentry/relay/pull/876))
```

Comment:
Missing mentioning ExpectCT reports , shouldn't we say: 
Support missing fields and directives for  ExpectCT and CSP reports ?

Reply:
You're right, let me split it up again.